crates/uv/tests/venv.rs

```diff
@@ -264,7 +264,7 @@ fn create_venv_unknown_python_minor() {
         ----- stdout -----
 
         ----- stderr -----
-          × No interpreter found for Python 3.100 in system toolchains
+          × No toolchain found for Python 3.100 in system path or `py` launcher
```

Comment:
What are your thoughts on "toolchain" as user-facing copy vs. interpreter? I feel like interpreter is more familiar to folks but seeing that you proactively changed it here.

Reply:
I'm not sure, but I'd like to be consistent. We're going to talk about managed toolchains a lot going forward so I think we might want to move in that direction.

A nuance is that we look for an interpreter in order to discover a toolchain. I guess that leans towards using "interpreter" here.

Reply:
I weakly prefer interpreter I think. It seems likely to land better with Python users. But the inconsistency with using "toolchain" in written docs does seem unfortunate. How crazy would it be to switch to "interpreter" everywhere?

Reply:
Well.. I talked about that in the initial design and @konstin did clarify that there's much more than the interpreter involved e.g. the managed toolchains include the standard library.

Reply:
For this change, I'll use "interpreter" because I want to get these changes out asap.

Reply:
Ah yeah I hear ya. If y'all already dug into the naming here then I defer to you. :)

Reply:
I have yet to be convinced that "toolchain" is the best user-facing concept :) we should all chat more.